pkg/secretutil/censor.go

```diff
@@ -108,11 +120,17 @@ func (c *ReloadingCensorer) Refresh(secrets ...string) {
 			// to justify secret booleans, since all values are known and replacing them carries a high price.
 			continue
 		}
+		if len(secret) < c.minimumSecretLength {
+			// Skip secrets that are shorter than the minimum length
+			continue
+		}
 
 		addReplacement(secret)
 		for _, item := range toEncode {
 			encoded := base64.StdEncoding.EncodeToString([]byte(item))
-			addReplacement(encoded)
+			if len(encoded) >= c.minimumSecretLength {
```

Comment:
Seems odd that a secret might be censored but the base64 encoded value would not - likely best to omit this check and let the length check on L123 suffice.